### ISSUE TITLE
Support multi-episodes files

### ISSUE THREAD

**Issue by zoriya** [zoriya/Kyoo#282]
### Feature description

Some files contains multiples episodes (ex Serie S01E23-24), some files contains only part of the episode (ex Seri S01E01 Part 1).
Those files should be handled somehow by kyoo.

(i have no idea how to actually handle those)

Part 1/2 could maybe be handled by the transcoder to merge them?
Episode metada could be changed in the middle of the episode on the front for multi-episode files?

**Comment by Eidenz** [comment]
This issue should be a priority. I believe I'm not the only user with lots of series having multi-episodes files, preventing me from using kyoo at all or requiring a config change in Sonarr just for this frontend.

**Comment by zoriya** [comment]
I agree that's pretty important but supporting this require non-trivial changes to the code. I don't even know how to show in the app that an episode is multi-part (open for suggestions).
